src/layers/icon-layer/icon-layer.js

```diff
@@ -97,30 +110,29 @@ export default class IconLayer extends Layer {
     };
   }
 
-  async getSvgIcons() {
```

Comment:
async is no longer required despite your fetch operation?

Reply:
no await used here